core/src/test/java/org/apache/calcite/test/RelBuilderTest.java

```diff
@@ -3362,4 +3364,89 @@ private void checkExpandTable(RelBuilder builder, Matcher<RelNode> matcher) {
             builder.literal(5));
     assertThat(call.toStringRaw(), is("BETWEEN ASYMMETRIC($0, 1, 5)"));
   }
+
+  @Test public void testSortExchangeFieldTrimmer() {
```

Comment:
I know that we have a lot of catch-all tests - but I think these testcases fall outside of the scope of `RelBuilderTest`

Reply:
Moved to `RelFieldTrimmerTest`